samples/snippets/batch_process_documents_sample_v1beta3.py

```diff
@@ -78,23 +78,24 @@ def batch_process_documents(
 
     for i, blob in enumerate(blob_list):
         # Download the contents of this blob as a bytes object.
-        blob_as_bytes = blob.download_as_bytes()
-        document = documentai.types.Document.from_json(blob_as_bytes)
-
-        print(f"Fetched file {i + 1}")
-
-        # For a full list of Document object attributes, please reference this page: https://googleapis.dev/python/documentai/latest/_modules/google/cloud/documentai_v1beta3/types/document.html#Document
-
-        # Read the text recognition output from the processor
-        for page in document.pages:
-            for form_field in page.form_fields:
-                field_name = get_text(form_field.field_name, document)
-                field_value = get_text(form_field.field_value, document)
-                print("Extracted key value pair:")
-                print(f"\t{field_name}, {field_value}")
-            for paragraph in document.pages:
-                paragraph_text = get_text(paragraph.layout, document)
-                print(f"Paragraph text:\n{paragraph_text}")
+        if ".json" in blob.name:
```

Comment:
What happens if it's not json?   What if the content of the file is valid json but the file doesn't have .json in it?   If you want it to exit if it's not json, it's a little smaller to add just a little verification line at the top, rather than wrap the whole thing in an "if" statement.  eg:

```
if ".json" not in blob.name:
   return 
```

another option is to wrap it in a try/ except -- so it always tries to parse the json, regardless of the filename